torch_geometric/nn/conv/rgcn_conv.py

```diff
@@ -59,6 +60,8 @@ class RGCNConv(MessagePassing):
         compensate.
         We advise to check out both implementations to see which one fits your
         needs.
+        RGCNConv can use dynamic shapes. If your device doesn't support dynamic
+        shapes, use FastRGCNConv.
```

Comment:
Could you elaborate on what you mean by `dynamic shapes`?

Reply:
Done.

Reply:
I am also not sure what you mean by this. Do you have a pointer or some explanation?

Reply:
Definition from https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#work_dynamic_shapes 
"Dynamic Shapes is the ability to defer specifying some or all tensor dimensions until runtime. "
For example You can see that torch.nonzero causes host synchronization for cuda device:
https://pytorch.org/docs/stable/generated/torch.nonzero.html

Reply:
Got it. Can we add a link to the NVIDIA doc here to make this more understandable for readers?

Reply:
Sure, added.